v2/oauth/multiple-frontends-with-separate-backends.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: Multiple frontend domains with separate backends
+hide_title: true
+---
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import CoreInjector from "/src/components/coreInjector"
+
+# Multiple frontend domains with separate backends
+
+You can use the following guide if you have a single [**Authentication Service**](/docs/oauth/introduction#authentication-service) that is used by multiple applications.
+In turn, each app has separate **`frontend `** and **`backend`** instances that are served from different domains.
+
+Using the actual **OAuth 2.0** terminology, we can say that each **`frontend`** can be considered a [**Client**](/docs/oauth/introduction#client) and each **`backend`** is a [**Resource Server**](/docs/oauth/introduction#resource-server).
```

Comment:
Hmm. I dont think so. The client here is both, the FE and the backend put together (from the point of view of the oauth provider). The resource server is the backend

Reply:
I agree with the first part (the client is both BE+FE) but the backend is not considered a resource server unless it has APIs  that take the OAuth2 access token. In this example I don't think it does that.

Reply:
aren't those backends serving resources? can you give a real world example of how this will look like?

Reply:
not necessarily, one example is our own ThirdParty integrations, where we do not actually serve any resources based on the access tokens issued by for example google. We just take those tokens, validate them and then issue our own session tokens.